admin/admin-domain-service/ballerina/apiCategoryDAO.bal

```diff
@@ -129,3 +129,18 @@ isolated function deleteAPICategoryDAO(string id, string org) returns commons:AP
         }
     }
 }
+
+isolated function getAPICountOfAPICategoryDAO(string apiCategory) returns int|commons:APKError {
+    postgresql:Client|error dbClient = getConnection();
+    if dbClient is error {
+        return e909401(dbClient);
+    } else {
```

Comment:
We can extract out this else block as well

Reply:
Fixed in https://github.com/wso2/apk/pull/1165/commits/453160d11adca7c2d3c310c822c32fc50e235f3a